File: Java/program26.java
////////////////////////////////////////////////////
//
//  File name   :   program26.java
//  Description :   Factorial
//  Input       :   5
//  Output      :   120
//  Author      :   Vaibhav Patil
//  Date        :   18/05/2025

import java.util.Scanner;

class Factorial
{
    public int Fact(int iNo)
    {
        int iCnt = 0;
        int iFact = 1;

        for(iCnt = 1;iCnt <= iNo;iCnt++)
        {
            iFact = iFact * iCnt;
        }
        return iFact;
    }
}
class program26
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        int iRet = 0;

        System.out.println("Enter the number: ");
        iValue = sobj.nextInt();

        Factorial dobj = new Factorial();
        iRet = dobj.Fact(iValue);

        System.out.println("Factorial is : "+iRet);
    }
}

File: program26.c
///////////////////////////////////////////////////////
//
//  File name :     program26.c
//  Descreption :   Factoria;
//  Author :        Vaibhav
//  Date :          11/05/2025
//
///////////////////////////////////////////////////////

#include<stdio.h>

int Factorial(int iValue)
{
    int iCnt = 0;
    int iFact = 1;

    for(iCnt = 1;iCnt <= iValue;iCnt++)
    {
        iFact = iFact * iCnt;
    }
    return iFact;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("_____________________________________________________________________\n");
    printf("--------- This program is used to do Addition ---------\n");
    printf("_____________________________________________________________________\n");

    printf("Enter a number : ");
    scanf("%d",&iValue);

    iRet = Factorial(iValue);

    printf("Factorial is : %d",iRet);
    printf("\n_____________________________________________________________________");
    return 0;
}